exercise/templates/exercise/exercise_plain.html

```diff
@@ -44,7 +44,7 @@
                     {% include 'exercise/_warnings_overlay.html' %}
                 {% endif %}
 
-                {% if exercise.is_submittable %}
+                {% if exercise.is_submittable and not exercisecollection %}
```

Comment:
If you reorder this, then we would have `if exercisecollection` and `elif exercise.is_submittable`.